app/scripts/modules/core/whatsNew/whatsNew.read.service.ts

```diff
@@ -25,16 +26,26 @@ export interface IWhatsNewContents {
 export class WhatsNewReader {
 
   private static extractFileContent(data: IGistApiResponse) {
-    return data.files[NetflixSettings.whatsNew.fileName].content;
+    const fileName = SETTINGS.feature.netflixMode ? NetflixSettings.whatsNew.fileName : SETTINGS.changelog.fileName;
+    return data.files[fileName].content;
   }
 
-  static get $inject() { return ['$http', '$log']; }
+  static get $inject() { return ['$http', '$log', '$q']; }
 
-  constructor(private $http: IHttpService, private $log: ILogService) {}
+  constructor(private $http: IHttpService, private $log: ILogService, private $q: IQService) {}
 
   public getWhatsNewContents(): IPromise<IWhatsNewContents> {
-    const gistId = NetflixSettings.whatsNew.gistId,
+    let gistId: string, accessToken: string;
+    if (SETTINGS.feature.netflixMode) {
+      gistId = NetflixSettings.whatsNew.gistId;
```

Comment:
I agree - the problem is handling the case in which we don't want to render any gist at all (since the Netflix `gistId` is in everyone's `settings.js`).